src/platform/site-wide/cta-widget/index.js

```diff
@@ -354,14 +355,30 @@ export class CallToActionWidget extends React.Component {
 
     if (!accountLevel) {
       return {
-        heading: `You’ll need to create a My HealtheVet account before you can ${
+        heading: `Please create a My HealtheVet account to ${
           this._serviceDescription
         }`,
-        buttonText: 'Create a My HealtheVet Account',
-        buttonHandler:
+        alertText: (
+          <>
+            <p>
+              You’ll need to create a My HealtheVet account before you can{' '}
+              {this._serviceDescription} online. This account is cost-free and
```

Comment:
The "online" part might be redundant due to the secondary description "schedule, reschedule, or cancel a VA appointment online" for appointments.